prisma/migrations/20240618114300_measurement_values/migration.sql

```diff
@@ -0,0 +1,235 @@
+----------------------------------------------------------------------------------------
+-- Migration to update existing tables with new taxa-based variables
+----------------------------------------------------------------------------------------
+
+----------------------------------------------------------------------------------------
+-- Add some new marking locations for caribou.
+----------------------------------------------------------------------------------------
+INSERT into lk_marking_type
+(name, description)
+VALUES
+(
+    'VHF collar',
+    'A collar that emits a radio signal detectable by a receiver, used for tracking the caribou location over shorter distances.'
+),
+(
+    'GPS collar',
+    'A collar that uses satellite technology to provide precise location data, which can be accessed remotely or when retrieved.'
+),
+(
+    'Satellite collar',
+    'A collar that transmits real-time location data via satellite, enabling continuous tracking of the caribou over large distances.'
+),
+(
+    'Sleeve',
+    'A "sleeve" in the context of animal marking is a durable, weather-resistant leg band or collar used to identify and track individual animals, such as caribou, without impeding their natural movement.'
+);
+INSERT into xref_taxon_marking_body_location
+(body_location, description, itis_tsn)
+VALUES
+(
+    'Neck',
+    NULL,
+    331030
+),
+(
+    'Left Forelimb (Anterior)',
+    'The front left limb.',
+    914181
+),
+(
+    'Right Forelimb (Anterior)',
+    'The front right limb.',
+    914181
+),
+(
+    'Left Hindlimb (Posterior)',
+    'The rear left limb',
+    914181
+),
+(
+    'Right Hindlimb (Posterior)',
+    'The rear right limb.',
+    914181
+);
+----------------------------------------------------------------------------------------
+-- Edit enum list to include some more units
+----------------------------------------------------------------------------------------
+
+ALTER TYPE critterbase.measurement_unit ADD VALUE 'rings';
```

Comment:
Did you mean to add these to the enum? I think these are 'measurement_name' properties.